src/Services/Metrics/Metrics.API/Program.cs

```diff
@@ -0,0 +1,38 @@
+using Metrics.API.Installers;
+using Metrics.Application;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
```

Comment:
I think these are middlewares in MovieInformation, but this is fine as well. No need to change